app/views/sign_up/passwords/new.html.erb

```diff
@@ -14,6 +14,7 @@
       html: { autocomplete: 'off' },
     ) do |f| %>
   <%= f.input :password, required: true,
+                         label: t('idv.form.password'),
```

Comment:
Looks like this is a string from IDV we're using on the generic signup page? What if we moved it to `forms/en.yml` instead of `idv/en.yml`?

Also is there a spec we can add in `/spec/views/sign_up/passwords/new.html.erb_spec.rb` to catch any regressions?